core/kazoo_stdlib/src/kz_json.erl

```diff
@@ -1425,3 +1425,9 @@ exec(Funs, ?JSON_WRAPPER(_)=JObj) ->
 exec_fold({F, K, V}, C) when is_function(F, 3) -> F(K, V, C);
 exec_fold({F, V}, C) when is_function(F, 2) -> F(V, C);
 exec_fold(F, C) when is_function(F, 1) -> F(C).
+
+-spec utf8_binary(any()) -> kz_term:ne_binary().
```

Comment:
Spec should be `utf8_binary(json_term()) -> json_term().` `any()` means I could pass a PID, Reference or Port value - those are not allowed.